### DISCUSSION TITLE
toDegrees float signature?

### DISCUSSION THREAD

**Discussion by ghost** [JOML-CI/JOML#343] [General]
https://github.com/JOML-CI/JOML/blob/52c7d151bd1fc13bdd5a0ecb9b97e97bbb41af82/src/main/java/org/joml/Math.java#L410-L419

`toRadians` may `return (float)`, but not `toDegrees`. Any particular reason for this? Precision? etc.

**Comment by httpdigest** [comment]
No particular reason. Just an oversight. Added now with https://github.com/JOML-CI/JOML/commit/9f67120a6c4f47408b33ae8298ad33c60eee78e7 . Thanks!
